### PR TITLE
fix: 资源池问题修复

### PR DESCRIPTION
**Pull request by iSecloud**

### CODE REVIEW

File: dbm-ui/backend/db_meta/models/cluster.py [hunk 1]
```diff
@@ -183,8 +183,8 @@ def get_partition_port(self):
             return self.proxyinstance_set.first().port
 
     @classmethod
-    def is_refer_deploy_plan(cls, deploy_plan_ids):
-        return cls.objects.filter(deploy_plan_id__in=deploy_plan_ids).exists()
+    def is_refer_deploy_plans(cls, deploy_plan_ids):
```
Comment (zhangzhw8) [inline dbm-ui/backend/db_meta/models/cluster.py:186]: 本来是返回 bool 值，方法名用 `is_` 开头没问题，但是现在改成了返回，也改一下方法名吧
`def get_cluster_deploy_plan_ids(cls, deploy_plan_ids: List[int]) -> List[int]`

File: dbm-ui/backend/db_meta/models/spec.py [hunk 2]
```diff
@@ -67,17 +69,25 @@ class ClusterDeployPlan(AuditedModel):
 
     name = models.CharField(max_length=128, default="")
     shard_cnt = models.PositiveIntegerField(default=0, help_text=_("集群分片总数"))
-    capacity = models.PositiveIntegerField(default=0, help_text=_("集群存储预估总容量/G"))
+    capacity = models.CharField(max_length=128, default="", help_text=_("集群存储预估总容量/G"))
```
Comment (zhangzhw8) [inline dbm-ui/backend/db_meta/models/spec.py:72]: 这里可能有变动，端午后跟产品对一下再做调整

File: dbm-ui/backend/db_meta/models/spec.py [hunk 3]
```diff
@@ -67,17 +69,25 @@ class ClusterDeployPlan(AuditedModel):
...
-    def get_apply_params_details(self, affinity=None, location_spec=None):
+    def get_apply_params_details(self, bk_cloud_id, affinity=None, location_spec=None):
         # 获取资源申请的参数，暂时忽略亲和性和位置参数过滤
-        backend_master_detail = self.spec.get_apply_params_detail(group_mark="master", count=self.machine_pair_cnt)
-        backend_slave_detail = self.spec.get_apply_params_detail(group_mark="slave", count=self.machine_pair_cnt)
-        return [backend_master_detail, backend_slave_detail]
+        backend_group_params = [
+            self.spec.get_apply_params_detail(
+                group_mark=f"backend_group_{group}",
+                count=2,
```
Comment (zhangzhw8) [inline dbm-ui/backend/db_meta/models/spec.py:83]: 这里一定是每组2台吗，如果后面出现每组 3 台的场景呢？
且暂先不考虑后续扩展的事情，至少这个 2 不应该以魔数出现，放到 constant 里面吧

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=153) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=153) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: iSecloud<br/>:x: kiozhang(张浩)<br/><hr/>**kiozhang(张浩)** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/blueking-dbm?pullRequest=153) it.</sub>

**Review by zhangzhw8 — APPROVED** [review]

**Pull request merged**